code/modules/overmap/ftl_shunt/computer.dm

```diff
@@ -146,3 +218,46 @@
 				return TOPIC_REFRESH
 		return TOPIC_REFRESH
 
+	if(href_list["adjust_power"])
+		var/input_power
+		input_power = input(user, "Enter allowed power input (in kilowatts)", "FTL Computer", (linked_core.allowed_power_usage / 1000)) as num|null
+		if(!input_power)
+			return
+
+		if(!CanInteract(user, state))
+			return TOPIC_NOACTION
+
+		input_power = input_power * 1000
+		linked_core.allowed_power_usage = clamp(input_power, 0, linked_core.max_power_usage)
+		return TOPIC_REFRESH
+
+	if(href_list["adjust_target"])
+		var/target_joules
+		target_joules = input(user, "Enter desired capacitor charge level (in megajoules)", "FTL Computer", (linked_core.target_charge / 1000)) as num|null
+		if(!target_joules)
+			return
+
+		if(!CanInteract(user, state))
+			return TOPIC_NOACTION
+
+		target_joules = target_joules * 1000
+		linked_core.target_charge = clamp(target_joules, 0, linked_core.max_charge)
+		return TOPIC_REFRESH
+
+	if(href_list["toggle_charge"])
+		linked_core.charging = !linked_core.charging
+
+	if(href_list["plot_jump"])
+		if(jump_plot_timer)
+			return // already plotting a jump.
+		var/jd = plot_jump(to_plot_x, to_plot_y)
+		ping("Jump plotting initiated, ETA [jd/20] seconds.")
```

Comment:
20?

Reply:
magic numbered until it seemed accurate enough

Reply:
But isn't `jd` literally the delay in deciseconds until the timer fires?